### PR TITLE
IS-16790

### PR DESCRIPTION
**Pull request by erik-leven**

### CODE REVIEW

File: templates/customer.json [hunk 1]
```diff
@@ -0,0 +1,244 @@
+[
+  {
+    "_id": "{{@ system @}}-{{@ datatype @}}-collect",
+    "comment": "WARNING! This pipe is generated from the template of the 'invoice' datatype and changes will be silently ignored during collapse. For more information see the connectorpy README.",
```
Comment (BaardBouvet) [inline templates/customer.json]: THis generated comment should be removed

File: templates/invoice.json [hunk 2]
```diff
@@ -91,7 +107,7 @@
                   [
                     "add",
                     "businessId",
-                    "{{@ account_id @}}"
+                    "QnVzaW5lc3M6NjJlZGM5ZmYtOWFkYi00OTBhLWJiYmUtOTI4ZDJiYzhiZjAz"
```
Comment (BaardBouvet) [inline templates/invoice.json]: This is wrong

**Review by BaardBouvet — CHANGES_REQUESTED** [review]
See comments inline

File: templates/invoice.json (hunk 2 above)
Comment (erik-leven) [reply templates/invoice.json]: hmm, I dont see that locally...

File: templates/customer.json (hunk 1 above)
Comment (erik-leven) [reply templates/customer.json]: fixed

File: templates/invoice.json (hunk 2 above)
Comment (erik-leven) [reply templates/invoice.json]: fixed

**Review by BaardBouvet — APPROVED** [review]

**Pull request merged**